postgresql/db-common-status-9-2.yml

```diff
@@ -0,0 +1,39 @@
+type: db
+data:
+  query: SELECT * FROM pg_stat_database;
+  dbUrl: jdbc:postgresql://${POSTGRESQL_DB_HOST_PORT}
+  dbDriverClass: org.postgresql.Driver
+  dbUser: ${POSTGRESQL_DB_USER}
+  dbPassword: ${POSTGRESQL_DB_PASSWORD}
+  dbAdditionalConnectionParams: ${POSTGRESQL_DB_ADDITIONAL_PARAMS}
+  #  dbVerticalModel="true" - because output of SHOW STATUS command gives each attribute in its own row
+  # (1st column being attribute name, 2nd column being attribute value)
+  dbVerticalModel: true
+
+require:
+  - className: com.sematext.spm.client.postgresql.PostgreSQLVersionCheck
+    values: 9.2
+
+observation:
+  - name: database
+    metricNamespace: postgresql
+
+    metric:
+      - name: database.deadlocks
+        source: deadlocks
+        type: long_counter
+        label: Deadlocks
+        description: Number of deadlocks detected in this database
+
+      - name: database.temp.bytes
+        source: temp_bytes
+        type: long_counter
+        unit: bytes
+        label: Temporary written bytes
+        description: Total amount of data written to temporary files by queries in this database. All temporary files are counted, regardless of why the temporary file was created, and regardless of the log_temp_files setting
+
+      - name: database.temp.files
+        source: temp_files
+        type: long_counter
+        label: Temporary files
+        description: Number of temporary files created by queries in this database. All temporary files are counted, regardless of why the temporary file was created (e.g., sorting or hashing), and regardless of the log_temp_files setting
```

Comment:
Check if the YAML is valid. If the description has special characters you might need to enclose inside double quotes

Reply:
I quote it, i also quote multiple line query and check all YAML file with https://codebeautify.org/yaml-validator